lib/commands/app/create.ts

```diff
@@ -37,16 +38,25 @@ export default class AppCreateCmd extends Command {
 
 		Create a new balena application.
 
-		You can specify the application device type with the \`--type\` option.
-		Otherwise, an interactive dropdown will be shown for you to select from.
+		You can specify the organization the application should belong to using
+		the \`--organization\` option. The organization's handle, not its name,
+		should be provided. Organization handles can be listed with the
+		\`balena orgs\` command.
 
-		You can see a list of supported device types with:
+		The application's default device type is specified with the \`--type\` option.
+		The \`balena devices supported\` command can be used to list the available
+		device types.
+
+		If no device type or organization is specified, interactive dropdowns
+		will be shown for selection (if there are multiple options to choose from).
+		If you are associated with only one organization, then that will be used by default,
+		and there is no need to specify it explicitly.
```

Comment:
Would the following suggestion "read a bit easier"?
```
Interactive dropdowns will be shown for selection if no device type or
organization is specified and there are multiple options to choose from.
If there is a single option to choose from, it will be chosen automatically.
This interactive behavior can be disabled by explicitly specifying a device
type and organization.
```

This suggestion purposely starts the paragraph with _"Interactive dropdowns will be shown for selection"_ and includes an explicit hint of how to disable it, so that it sounds alarming and useful to anyone scripting the CLI and skimming through the help output.

Otherwise, nit: One of the original lines is longer than 80 characters.